features/popover-hint.yml

```
@@ -0,0 +1,4 @@
+name: Popover=hint
+description: The `hint` value for the `popover` HTML attribute creates a popover that is "subordinate" to `popover=auto` popovers. Therefore, `popover=hint`'s will not close other `popover=auto`'s.
```

Some of this is a style suggestion to be more consistent with the way we've written other descriptions. But I'm also trying to see if we can describe what this does for developers, rather than an abstract relationship of values. What do you think of this?

```suggestion
description: The `popover="hint"` global HTML attribute creates a popover that is subordinate to popovers with a `popover="auto"` attribute. You can use this to create tooltips that don't dismiss `auto` popovers.
```

---

Sounds perfect to me!